ssm/hmm.py

```diff
@@ -42,9 +42,11 @@ def __init__(self, K, D, M=0, init_state_distn=None,
         # Make the initial state distribution
         if init_state_distn is None:
             init_state_distn = isd.InitialStateDistribution(K, D, M=M)
-        if not isinstance(init_state_distn, isd.InitialStateDistribution):
+        if not (isinstance(init_state_distn, isd.InitialStateDistribution) \
+            or isinstance(init_state_distn, isd.FixedInitialStateDistribution)):
```

Comment:
I think this is unnecessary as `FixedInitialStateDistribution` inherits from `InitialStateDistribution`.  They are both instances of `InitialStateDistribution`.

Reply:
Good point, thanks Scott!